app/Console/Commands/GenerateLateFeeCommand.php

```diff
@@ -15,7 +15,7 @@ class GenerateLateFeeCommand extends BaseCommand
      *
      * @var string
      */
-    protected $signature = 'make:late-fee' . '{--day= : Generate late fee for a specific day of the month in the past e.g 18. Must be in range 1 - 31} ';
+    protected $signature = 'make:late-fee-gen' . '{--day= : Generate late fee for a specific day of the month in the past e.g 18. Must be in range 1 - 31} ';
```

Comment:
why are we changing this again?

Reply:
it is clashing with make:late-fee-debit
one is to generate, other is to debit